doc/Tribler REST API.md

```diff
@@ -127,6 +127,36 @@ Returns all discovered channels in Tribler.
 }
 ```
 
+## `PUT /channels/discovered/{channelcid}`
```

Comment:
This makes no sense to me. By going to the discovered channels with a channel id parameter I create my own channel? What if I use a channel id already in use by someone else? This endpoint is confusing imo.